lib/payoneer/configuration.rb

```diff
@@ -5,14 +5,22 @@ class Configuration
     DEVELOPMENT_API_URL = 'https://api.sandbox.payoneer.com/Payouts/HttpApi/API.aspx?'
     PRODUCTION_API_URL = 'https://api.payoneer.com/Payouts/HttpApi/API.aspx?'
 
-    attr_accessor :environment, :partner_id, :partner_username, :partner_api_password
+    attr_accessor :environment, :partner_id, :partner_username, :partner_api_password, :auto_approve_sandbox_accounts
 
     def initialize
       @environment = DEVELOPMENT_ENVIRONMENT
     end
 
+    def development?
+      environment == DEVELOPMENT_ENVIRONMENT
+    end
+
     def api_url
-      environment == PRODUCTION_ENVIRONMENT ? PRODUCTION_API_URL : DEVELOPMENT_API_URL
+      development? ? DEVELOPMENT_API_URL : PRODUCTION_API_URL
+    end
+
+    def auto_approve_sandbox_accounts?
+      development? && !!auto_approve_sandbox_accounts
```

Comment:
why not just set `@auto_approve_sandbox_accounts = false` in initialize? That way the double negative can be removed